spec/classes/letsencrypt_spec.rb

```diff
@@ -211,6 +211,41 @@
             end
           end
 
+          describe 'renew_cron_ensure and disable_distro_cron (with systemd)' do
+            let(:additional_params) do
+              { renew_cron_ensure: 'present' }
+            end
+            let(:facts) do
+              facts.merge({
+                            service_provider: 'systemd',
+                          })
+            end
+
+            it do
+              case facts[:osfamily]
+              when 'RedHat'
+                is_expected.to contain_service('certbot-renew.timer').with(ensure: 'stopped', enable: false)
+              when 'Debian'
+                is_expected.to contain_service('certbot.timer').with(ensure: 'stopped', enable: false)
+              end
+            end
+          end
+
+          describe 'renew_cron_ensure and disable_distro_cron (without systemd)' do
+            let(:additional_params) do
+              { renew_cron_ensure: 'present' }
+            end
+
+            it do
+              case facts[:osfamily]
```

Comment:
can you please don't use the legacy facts?
```suggestion
              case facts[:os]['family']
```

Reply:
Done